components/MainMenuPage/Navbar/index.js

```diff
@@ -3,28 +3,27 @@ import PlayButton from "../NavbarComponents/Play";
 
 export default function Navbar() {
   return (
-    <naStyledNavv>
+    <StyledNav>
       <StyledUl>
         <StyledLi>
```

Comment:
Why do you need a list and a `li` for only one element inside the list?

If you keep it (but I don't see why), please add `role="list"` to the `ul` in order [to ensure a11y in Safari as well](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type#accessibility_concerns).

Reply:
Actually, there are other elements in the navbar that follow, I've already anticipated them